include/zephyr/dt-bindings/input/input-event-codes.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_INPUT_INPUT_EVENT_CODES_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_INPUT_INPUT_EVENT_CODES_H_
+
+/* Input event types. */
```

Comment:
It would be nice to use Doxygen here, that is:

- Add it to the input API group (as a sub-group)
- Group definitions using `@name`, potentially use anchors to reference

See an example here: https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/dt-bindings/regulator/npm6001.h

Reply:
Cool idea, done, lmk if I got it right.